### PR TITLE
Samples from bigquery-samples-python

### PR DESCRIPTION
**Pull request by elibixby**
PTAL @GoogleCloudPlatform/devrel-samples

### CODE REVIEW

File: bigquery/tox.ini [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[tox]
```
Comment (tmatsuo) [inline bigquery/tox.ini]: I will merge necessary bit to the top level tox.ini, so maybe you can remove it?

File: bigquery/dump/load-data-by-POST.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
+import sys
```
Comment (tmatsuo) [inline bigquery/dump/load-data-by-POST.py]: Isn't the 'dump' dir used for just copying something outside to the repo?
Do we need this file?

Comment (elibixby) [inline bigquery/dump/load-data-by-POST.py]: I was using dump as a place to put files that were included in docs but I hadn't had time to clean up yet. I can delete it.

**Comment by proppy** [comment]
It's probably a good idea to remove `-` (hyphen) from the name, it's not common in python and make itu un-importable.

**Comment by tmatsuo** [comment]
Ok, LGTM
Just a nit. I would like anyone to wait for the reviewers explicitly say LGTM before merging

**Comment by elibixby** [comment]
Yup. Sorry got ahead of myself.

**Pull request merged**